tests/functional/cylc-reinstall/01-file-transfer.t

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#------------------------------------------------------------------------------
+# Test workflow re-installation file transfer
+. "$(dirname "$0")/test_header"
+set_test_number 14
+
+# Test cylc install copies files to run dir successfully.
+TEST_NAME="${TEST_NAME_BASE}-basic"
+make_rnd_suite
+pushd "${RND_SUITE_SOURCE}" || exit 1
+mkdir .git .svn dir1 dir2-be-removed 
+touch .git/file1 .svn/file1 dir1/file1 dir2-be-removed/file1 file1 file2
+run_ok "${TEST_NAME}" cylc install "${RND_SUITE_NAME}"
+# If rose-cylc plugin is installed add install files to tree.
+export ROSE_FILES=''
+if python -c "import cylc.rose" > /dev/null 2>&1; then
+    export ROSE_FILES="├── opt
+│   └── rose-suite-cylc-install.conf
+└── rose-suite.conf"
+else
+    export ROSE_FILES=""
+fi
+
+tree -a -v -I '*.log|01-file-transfer*' --charset UTF8 --noreport "${RND_SUITE_RUNDIR}/run1" > '01-file-transfer-basic-tree.out'
+
+cmp_ok '01-file-transfer-basic-tree.out'  <<__OUT__
+${RND_SUITE_RUNDIR}/run1
+├── .service
+├── dir1
+│   └── file1
+├── dir2-be-removed
+│   └── file1
+├── file1
+├── file2
+├── flow.cylc
+├── log
```

Comment:
This and similar bits of tree output aren't safe if `cylc-rose` is not installed.
I created https://github.com/datamel/cylc-flow/pull/12 with a fix.
The problem is that if `${ROSE_FILES}==''`, then this will leave:
```
├── log
│   └── install
```
Where you'd want:
```
└── log
    └── install
```

Reply:
Thanks!